client.go

```diff
@@ -27,6 +28,18 @@ const (
 	defaultZkRoot        = "/hbase"
 	defaultZkTimeout     = 30 * time.Second
 	defaultEffectiveUser = "root"
+
+	// variables here are used for testing the Marshalling interface
+	ClientTypeJsonKey          = "ClientType"
+	ClientRegionMapJsonKey     = "ClientRegionMap"
+	RegionInfoMapJsonKey       = "RegionInfoMap"
+	KeyRegionCacheJson         = "KeyRegionCache"
+	ClientRegionCacheJsonKey   = "ClientRegionCache"
+	MetaRegionInfoJsonKey      = "MetaRegionInfo"
+	AdminRegionInfoJsonKey     = "AdminRegionInfo"
+	Done_StatusJsonKey         = "Done_Status"
+	RegionLookupTimeoutJsonKey = "RegionLookupTimeout"
+	RegionReadTimeoutJsonKey   = "RegionReadTimeout"
```

Comment:
Remove them. Just use the plain string in the test if we're checking for those json elements